AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5118] SG2 | Review and resolve Backstop Reference and Chromy errors

## Ticket(s)

**Jira Ticket**
- [EWL-5118: SG2 | Review and resolve Backstop Reference and Chromy errors](https://issues.ama-assn.org/browse/EWL-5118)

    ***Child Tickets***
    - [EWL-5067: SG2 | Update Backstop references for Tools List](https://issues.ama-assn.org/browse/EWL-5067)
    - [EWL-5130: SG2 | Fix visual regression in checkbox atom](https://issues.ama-assn.org/browse/EWL-5130)
    - [EWL-5081: SG2 | Review and resolve Backstop references - News and Topics](https://issues.ama-assn.org/browse/EWL-5081)

## Description
This update corrects all backstop reference images and regressions from latest `develop` branch. Regressions have been logged as a separate ticket detailing the regression and included in this PR.

Chromy errors were resolved with updates to backstop test cases that were not able to find the correct css selectors or pattern lab pages.


## To Test
- Pull `bugfix/EWL-5118-backstop-reference-chromy-errors` branch
- Run gulp serve in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Run `backstop test` and confirm no errors occur.

## Visual Regressions

Updates to reference images have been included in this PR.


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
